solr/solrj/src/java/org/apache/solr/common/params/CollectionParams.java

```diff
@@ -42,31 +42,30 @@
 
 
   enum LockLevel {
-    CLUSTER(0),
-    COLLECTION(1),
-    SHARD(2),
-    REPLICA(3),
-    NONE(10);
-
-    public final int level;
-
-    LockLevel(int i) {
-      this.level = i;
+    NONE(10, null),
```

Comment:
I don't think it makes sense to reorder them here.

Reply:
Compiler complained of forward reference when I didn't.

Reply:
Didn't consider that; yea, that's a good reason for reordering.